### ISSUE TITLE
Problem with fitting nodecov example on flomarriage from NME Day 2

### ISSUE THREAD

**Issue by smjenness** [statnet/statnetWeb#54]
This is with the CRAN version running locally, here is the error message under the hood.
```
Warning in strsplit(msg, "\n") :
  input string 1 is invalid in this locale
Error in parse(text = x, keep.source = FALSE) : <text>:1:21: unexpected input
1: nw() ~ edges+nodecov(�
```

**Comment by martinamorris** [comment]
I can't replicate this on my laptop


> aaa <- ergm(flomarriage~edges + nodecov("wealth"))
> aaa

MLE Coefficients:
          edges  nodecov.wealth
       -2.59493         0.01055

> sessionInfo()
R version 3.2.0 (2015-04-16)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 8 x64 (build 9200)

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United 
States.1252
[3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C
[5] LC_TIME=English_United States.1252

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] ergm_3.4.0           network_1.12.0       statnet.common_3.2.2

loaded via a namespace (and not attached):
  [1] DEoptimR_1.0-2    Matrix_1.2-1      parallel_3.2.0    tools_3.2.0 
coda_0.17-1
  [6] grid_3.2.0        robustbase_0.92-4 trust_0.1-6       lpSolve_5.6.11 
lattice_0.20-31

On Tue, 8 Sep 2015, Samuel Jenness wrote:

> 
> This is with the CRAN version running locally, here is the error message under the hood.
> 
> Warning in strsplit(msg, "\n") :
>   input string 1 is invalid in this locale
> Error in parse(text = x, keep.source = FALSE) : <text>:1:21: unexpected input
> 1: nw() ~ edges+nodecov(???
> 
> —
> Reply to this email directly or view it on GitHub.[AH0J4gKzyea4BfDxPQ9XprXBBW9rf5TEks5ovpUYgaJpZM4F5VQd.gif]
> 
> 
>

****************************************************************
  Professor of Sociology and Statistics
  Director, UWCFAR Sociobehavioral and Prevention Research Core
  Box 354322
  University of Washington
  Seattle, WA 98195-4322

  Office:        (206) 685-3402
  Dept Office:   (206) 543-5882, 543-7237
  Fax:           (206) 685-7419

morrism@u.washington.edu
http://faculty.washington.edu/morrism/

**Comment by smjenness** [comment]
Hi Martina - you do not have statnetWeb attached. This is a problem with statnetWeb, not ergm.

**Comment by martinamorris** [comment]
k.

Emily -- this is a reminder that making changes to a package before a 
workshop must be followed by A COMPLETE TEST OF ALL WORKSHOP EXAMPLES


On Tue, 8 Sep 2015, Samuel Jenness wrote:

> 
> Hi Martina - you do not have statnetWeb attached. This is a problem with statnetWeb, not ergm.
> 
> —
> Reply to this email directly or view it on GitHub.[AH0J4g5F8-vbuYOpajTRqhz5CZ2gPBPjks5ovtSigaJpZM4F5VQd.gif]
> 
> 
>

****************************************************************
  Professor of Sociology and Statistics
  Director, UWCFAR Sociobehavioral and Prevention Research Core
  Box 354322
  University of Washington
  Seattle, WA 98195-4322

  Office:        (206) 685-3402
  Dept Office:   (206) 543-5882, 543-7237
  Fax:           (206) 685-7419

morrism@u.washington.edu
http://faculty.washington.edu/morrism/

**Comment by martinamorris** [comment]
IIRC, Em updated the shiny server version of statnetWeb, but not CRAN. 
I'm not replicating the error using the shiny sever.

On Tue, 8 Sep 2015, Samuel Jenness wrote:

> 
> This is with the CRAN version running locally, here is the error message under the hood.
> 
> Warning in strsplit(msg, "\n") :
>   input string 1 is invalid in this locale
> Error in parse(text = x, keep.source = FALSE) : <text>:1:21: unexpected input
> 1: nw() ~ edges+nodecov(???
> 
> —
> Reply to this email directly or view it on GitHub.[AH0J4gKzyea4BfDxPQ9XprXBBW9rf5TEks5ovpUYgaJpZM4F5VQd.gif]
> 
> 
>

****************************************************************
  Professor of Sociology and Statistics
  Director, UWCFAR Sociobehavioral and Prevention Research Core
  Box 354322
  University of Washington
  Seattle, WA 98195-4322

  Office:        (206) 685-3402
  Dept Office:   (206) 543-5882, 543-7237
  Fax:           (206) 685-7419

morrism@u.washington.edu
http://faculty.washington.edu/morrism/

**Comment by smjenness** [comment]
Yes, the shinyapps.io kept timing out (not sure why, could be poor internet at the hotel) so we had to use the CRAN version in the workshop.

**Comment by martinamorris** [comment]
:{

On Tue, 8 Sep 2015, Samuel Jenness wrote:

> 
> Yes, the shinyapps.io kept timing out (not sure why, could be poor internet at the hotel) so we had to use the CRAN
> version in the workshop.
> 
> —
> Reply to this email directly or view it on GitHub.[AH0J4hTvA3Ok7ZqCL2I3dXA-Rv_8rUY4ks5ovtuXgaJpZM4F5VQd.gif]
> 
> 
>

****************************************************************
  Professor of Sociology and Statistics
  Director, UWCFAR Sociobehavioral and Prevention Research Core
  Box 354322
  University of Washington
  Seattle, WA 98195-4322

  Office:        (206) 685-3402
  Dept Office:   (206) 543-5882, 543-7237
  Fax:           (206) 685-7419

morrism@u.washington.edu
http://faculty.washington.edu/morrism/

**Comment by ebey** [comment]
I am also unable to recreate this error. 

Was that error message from the command line or one of the output sections in the app? 
Can you copy/paste the output from each of these sections when you get the error: ergm formula, summary statistics, current model summary

**Comment by smjenness** [comment]
It was generated in both the command line and visible within the app itself here:
<img width="616" alt="screen shot 2015-09-08 at 11 02 02 pm" src="https://cloud.githubusercontent.com/assets/2521570/9747410/acfc16d0-567d-11e5-8aa3-a262c250aae3.png">

**Comment by smjenness** [comment]
Hmmm... I wonder if this is how Safari (my web browser) is handling the quotation marks?
